integrations/pom.xml

```diff
@@ -12,9 +12,9 @@
   <name>Configuration as Code Integration tests</name>
 
   <properties>
-    <jenkins.version>2.176.3</jenkins.version> <!-- Remove when parent base is updated. -->
     <maven.deploy.skip>true</maven.deploy.skip> <!-- no need to be deployed during release, this is a test-only module -->
     <jackson.version>2.10.2</jackson.version>
+    <surefire.rerunFailingTestsCount>4</surefire.rerunFailingTestsCount>
```

Comment:
would it be possible to just rerun the test we know is flakey? (i.e. this is hiding the problem for all tests)

Reply:
As far as I know this property is applied to all tests, i. e., any failing test will be re-executed